akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolInterfaceActor.scala

```diff
@@ -26,7 +27,7 @@ private object PoolInterfaceActor {
 
   case object Shutdown extends DeadLetterSuppression
 
-  val name = new SeqActorName("PoolInterfaceActor")
+  val names = SeqActorName("PoolInterfaceActor")
```

Comment:
I’m slightly in favor of the singular form, but in any case it should be used uniformly.